packages/feedback/stories/feedback.stories.tsx

```diff
@@ -1,156 +1,157 @@
 import React from 'react';
-import { Story, Meta } from '@storybook/react';
+import { StoryObj } from '@storybook/react';
 
 import Feedback from '../src/Feedback';
 import FeedbackForm from '../src/FeedbackForm';
 // import README from '../README.md';
 
 export default {
   title: 'Components/Feedback',
-  parameters: {
-    docs: {
-      // page: README,
-    },
-  },
-} as Meta;
+  components: { Feedback, FeedbackForm },
```

Comment:
every component should be it's own stories file so it gets its own generated props table. Just add to the title for each, so `Components/Feedback/Feedback` & `Components/Feedback/FeedbackForm`